src/sample_submission_data.R

```diff
@@ -0,0 +1,9 @@
+## Script that generates sample submission data to pass validation checks
+require(arrow) # for parquet writing
+source('sim_model_output.R')
+# Function found in /src to generate data frame
+df_sim <- sim_model_output()
+
+# Note: make sure to point to the correct path for your data, defaults to folder 'model-output'
+arrow::write_parquet(df_sim, "auxiliary-data/example-files/2024-10-02-umass-validtestsubmission.parquet")
+hubValidations::validate_model_data(".", "2024-10-02-umass-validtestsubmission.parquet")
```

Comment:
`here::here()` points to the root of the hub, but I'm not sure this will work exactly since `validate_model_data()` expects your model file to be in the `model-outputs` directory. 

```suggestion
hubValidations::validate_model_data(here::here(), "2024-10-02-umass-validtestsubmission.parquet")
```

Reply:
Does `here::here()` point to the root of the hub or the root of the Rproj directory, which is in `src/`? My workflows of how to set things up have been a bit turned on their heads by the inclusion of the Rproj file in a subdirectory.

Reply:
Looking at the `here` documentation a bit, I've convinced myself that the `here_i_am()` call proposed at the beginning would establish `here::here()` as the `src/` directory. Which might then suggest that we need to replace `here::here()` in the above suggestion with `here::here("../")`, right?

Reply:
> I've convinced myself that the `here::i_am()` call proposed at the beginning would establish `here::here()` as the `src/` directory.

Nope. The `here::i_am("src/sample_submission_data.R")` statement is saying "This is a file whose path is `src/sample_submission_data.R`, _relative to the root of the project_, thus implying the root of the project. 

You can see this in [`src/make_round_config.R#L11`](https://github.com/reichlab/variant-nowcast-hub/blob/a9bc30382c29740f3447499acc64743e083aaf37/src/make_round_config.R#L11) defining `here::i_am("src/make_round_config.R")` and then [`src/make_round_config.R#L11`](https://github.com/reichlab/variant-nowcast-hub/blob/a9bc30382c29740f3447499acc64743e083aaf37/src/make_round_config.R#L203) defining `hub_root <- here::here()`